deploy/ansible/roles/nfs-server-setup/tasks/configure.yml

```
@@ -0,0 +1,41 @@
+---
+
+- name: Ensure the NFS utils package is installed
+  package:
+    name: nfs-utils
+    state: present
+
+- name: Ensure NFS Server is running
+  systemd:
+    name: nfsserver
+    state: started
+    enabled: yes
+
+- name: Ensure expected base directories are present
+  file:
+    path: "{{ item }}"
+    state: directory
+    mode: +r
+  with_items:
+    - "{{ target_media_location }}"
+    - "/usr/sap/{{ app_sid | upper }}/SYS"
+    - "/tmp/app_template"
+    - "/sapmnt/{{ app_sid | upper }}/global"
+    - "/sapmnt/{{ app_sid | upper }}/profile"
```

With the above changes to the `nfs_exports` default we could do:

```suggestion
- name: Ensure expected base directories are present
  file:
    path: "{{ item.path }}"
    state: directory
    mode: +r
  loop: "{{ nfs_exports }}"
```